lib/toolchain.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
-TAP_PATH = (Tap::TAP_DIRECTORY/"MaterializeInc"/"homebrew-crosstools").freeze
+TAP_DIR = HOMEBREW_TAP_DIRECTORY || Tap::TAP_DIRECTORY
```

Comment:
I don't think the `||` will work as you expect. `HOMEBREW_TAP_DIRECTORY` will just explode if it's not defined, right? I'd just use _only_ `HOMEBREW_TAP_DIRECTORY` and let's just tell everyone to upgrade.

Reply:
No idea! Using just `HOMEBREW_TAP_DIRECTORY` and telling folks to upgrade works for me!

Reply:
Ahhh yeah looks like you're right, it just explodes, [Playground](https://try.ruby-lang.org/playground/#code=FOO+%3D+%22hello%22%0ABAR+%3D+BAZ+%7C%7C+FOO%0Aprint+FOO&engine=cruby-3.2.0)